RNReanimated.podspec

```diff
@@ -104,9 +110,9 @@ Pod::Spec.new do |s|
     "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_TARGET_SRCROOT)\" \"$(PODS_ROOT)/#{folly_prefix}Folly\" \"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/Headers/Private/React-Core\"",
     "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
   }
-  s.compiler_flags = folly_compiler_flags + ' ' + boost_compiler_flags
+  s.compiler_flags = folly_compiler_flags + ' ' + boost_compiler_flags + ' -DHERMES_ENABLE_DEBUGGER'
```

Comment:
Do we really want to set `-DHERMES_ENABLE_DEBUGGER` even in release mode?

Reply:
This is the same as in React Native https://github.com/facebook/react-native/blob/68f3a42fc7380051714253f43b42175de361f8bd/ReactCommon/hermes/React-hermes.podspec#L40